### PR TITLE
feat: floor attenuation value

### LINKED ISSUE #69: Results should be rounded to the closest integer

### PR DESCRIPTION
**Pull request by dansvend**
Closes #69

### CODE REVIEW

**Review by tormodAase — APPROVED** [review]
Looks good! We could also consider saving the raw value, but displaying the floored value to users. This would give us more accurate logs, and at the same time give the user a simplified view of the data

**Comment by dansvend** [comment]
> Looks good! We could also consider saving the raw value, but displaying the floored value to users. This would give us more accurate logs, and at the same time give the user a simplified view of the data

Very good feedback! I agree. I've made changes so that we display floored attenuation values, but log floats 💪

File: contexts/_internal/useFlooredDecibelDifferenceResult.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    if (isValue(left)) {
+      left = createValue(Math.floor(unwrap(left)));
+    }
+    if (isValue(right)) {
+      right = createValue(Math.floor(unwrap(right)));
+    }
+    return { left, right };
+  }, [decibelDifferenceResult]);
+
+  return { flooredDecibelDifferenceResult };
```
Comment (tormodAase) [inline contexts/_internal/useFlooredDecibelDifferenceResult.ts:20]: If we have a utility function
```ts
export const getFlooredResult = (resultForASingleEar: CalculateDecibelDifferenceResult): CalculateDecibelDifferenceResult => {
    if (isValue(resultForASingleEar)) {
        return createValue(Math.floor(unwrap(resultForASingleEar)));
    }
    return resultForASingleEar;
}
```

Then we can simplify this part.

```suggestion
const flooredDecibelDifferenceResult = useMemo(() => ({
    left: getFlooredResult(decibelDifferenceResult.left),
    right: getFlooredResult(decibelDifferenceResult.right)
}), [decibelDifferenceResult])
```

Comment (dansvend) [reply contexts/_internal/useFlooredDecibelDifferenceResult.ts]: Good suggestion! I've now implemented the changes these changes 💪

**Review by tormodAase — APPROVED** [review]

**Pull request merged**